docs/source/release-notes.rst

```diff
@@ -3,6 +3,160 @@ FiftyOne Release Notes
 
 .. default-role:: code
 
+.. _release-notes-v0.8.0:
+
+FiftyOne 0.8.0
+--------------
+*Released March 31, 2021*
+
+App
+^^^
+- Added the ability to tag samples and labels directly from the App in both
+  the sample grid (macro) and expanded sample view (micro) with respect to and
+  filters or currently selected samples/labels
+- Added a `LABEL TAGS` section to the Filters Sidebar to coincide with the
+  introduction of label tags
+- Added label tooltips that display on hover in the expanded sample view
+- Expanded actions to list of button groups in the sample grid and expanded
+  sample view
+- Added support for rendering semantic labels in the new tooltip in the expanded
+  sample view for :class:`Segmentation <fiftyone.core.labels.Segmentation>`
+  mask values (pixel values) using the new
+  :attr:`Dataset.mask_targets <fiftyone.core.dataset.Dataset.mask_targets>`
+  and
+  :attr:`Dataset.default_mask_targets <fiftyone.core.dataset.Dataset.default_mask_targets>`
+  fields
+- Fixed hiding, clearing, and only showing selected samples in the samples grid
+
+Brain
+^^^^^
+- Added a :meth:`compute_visualization() <fiftyone.brain.compute_visualization>` method that uses embeddings and dimensionality reduction methods to generate interactive visualizations of the samples and/or labels in a dataset. Check out :ref:`this user guide page <XXXXXXXX>` for details. Features include:
```

Comment:
FYI- for any OCD folks out there, these lines intentionally don't have line breaks at 80 characters. If they do, it somehow messes up the parsing of the nested lists.

In fact, RST format actually wants nested lists to have newlines before and after them (and this is how nested lists in docstrings are written), but something about the style sheet for the HTML pages causes the indentation to be gigantic for these nested lists.

Instead, it seems that *not* including the newlines will still produce nested lists, but with smaller indents, which looks better in HTML format.